TestScene/TestSprite.cs

```diff
@@ -25,5 +26,12 @@ public override void _Ready()
 		GD.Print(storyEngine.CurrentNodeTeaserText());
 		GD.Print(storyEngine.CurrentNodeEventText());
 		GD.Print(storyEngine.CurrentNodeChoiceText(0));
+
+        GD.Print("\n\n");
+        string json = StoryEngineAPI.GetTestStoryJSON();
+        GD.Print(json);
+		StoryDataModel? story = StoryEngineAPI.DeserializeStoryFromJSON(json);
+        GD.Print("\n\n");
+		//if (story != null) GD.Print(StoryEngineAPI.SerializeStoryToJSON(story));
```

Comment:
Was the commented out line for testing?

Reply:
Ah yeah, I was testing deserializing most recently so that's probably exactly it.